Marlin/src/gcode/motion/G2_G3.cpp

```diff
@@ -103,7 +103,15 @@ void plan_arc(
               mm_of_travel = linear_travel ? HYPOT(flat_mm, linear_travel) : ABS(flat_mm);
   if (mm_of_travel < 0.001f) return;
 
-  uint16_t segments = FLOOR(mm_of_travel / (MM_PER_ARC_SEGMENT));
+  const feedRate_t scaled_fr_mm_s = MMS_SCALED(feedrate_mm_s);
+
+  #ifdef ARC_SEGMENTS_PER_SEC
+    float seg_length = scaled_fr_mm_s * _RECIP(ARC_SEGMENTS_PER_SEC);
```

Comment:
Usage of private define "_RECIP" from another module.

I have compilation fail here:
`Compiling .pio\build\LPC1768\src\src\gcode\motion\G2_G3.cpp.o
Marlin\src\gcode\motion\G2_G3.cpp: In function 'void plan_arc(const xyze_pos_t&, const ab_float_t&, uint8_t)':
Marlin\src\gcode\motion\G2_G3.cpp:109:41: error: '_RECIP' was not declared in this scope
     float seg_length = scaled_fr_mm_s * _RECIP(ARC_SEGMENTS_PER_SEC);
                                         ^~~~~~
*** [.pio\build\LPC1768\src\src\gcode\motion\G2_G3.cpp.o] Error 1`

Reply:
Is in possible to use RECIPROCAL() macro instead?